shell/platform/darwin/ios/framework/Source/accessibility_bridge_ios.h

```diff
@@ -24,6 +24,7 @@ class AccessibilityBridgeIos {
   virtual void DispatchSemanticsAction(int32_t id,
                                        flutter::SemanticsAction action,
                                        std::vector<uint8_t> args) = 0;
+  virtual void AccessibilityFocusDidChange(int32_t focused_id) = 0;
```

Comment:
Please add a docstring, what focused_id is could be misinterpreted (it could mean the old value).